app/controllers/newsletters/search.js

```diff
@@ -0,0 +1,101 @@
+import Controller from '@ember/controller';
+import moment from 'moment';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+
+export default class NewsletterInfosSearchController extends Controller {
+  @service router;
+  @service currentSession;
+
+  queryParams = {
+    searchText: {
+      type: 'string',
+    },
+    mandatees: {
+      type: 'string',
+    },
+    dateFrom: {
+      type: 'string',
+    },
+    dateTo: {
+      type: 'string',
+    },
+    page: {
+      type: 'number',
+    },
+    size: {
+      type: 'number',
+    },
+    sort: {
+      type: 'string',
+    },
+  };
+
+  sizeOptions = Object.freeze([5, 10, 20, 50, 100, 200]);
+
+  @tracked searchText = '';
+  @tracked mandatees;
+  @tracked dateFrom;
+  @tracked dateTo;
+  @tracked searchTextBuffer = '';
+  @tracked mandateesBuffer;
+  @tracked dateFromBuffer;
+  @tracked dateToBuffer;
+  @tracked popoverShown; // TODO, this is for a tooltip, this should be handled elsewhere
+  @tracked page;
+  @tracked size;
+  @tracked sort;
+  @tracked emptySearch;
+
+  constructor() {
+    super(...arguments);
+    this.page = 0;
+    this.size = this.sizeOptions[2];
+    this.sort = '-agendaitems.meetingDate';
+  }
+
+  deserializeDate(date) {
+    return date && moment(date, 'DD-MM-YYYY').toDate();
+  }
```

Comment:
Move to a util like `deserialize-date` that can be reused in all search routes?

Start using `parse` of https://date-fns.org/v2.29.1/docs/parse#usage instead of `moment`